src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const tabHead = document.querySelector('thead');
+const tabBody = document.querySelector('tbody');
+
+tabHead.addEventListener('click', (events) => {
```

Comment:
```suggestion
tabHead.addEventListener('click', (event) => {
```
You handle one event at a time, let's not confuse other developers